criu/cr-restore.c

```diff
@@ -2192,6 +2192,29 @@ static int write_restored_pid(void)
 	return 0;
 }
 
+static void reap_zombies(pid_t child_pid) {
+	pid_t current_pid;
+	int current_status;
+
+	while (1) {
+		current_pid = waitpid(-1, &current_status, WNOHANG);
```

Comment:
I don't think that we need WNOHANG here. Otherwise it will be heating the planet;).